exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -45,10 +66,33 @@ type prwExporter struct {
 	retrySettings     configretry.BackOffConfig
 	wal               *prweWAL
 	exporterSettings  prometheusremotewrite.Settings
+	telemetry         prwTelemetry
+}
+
+func newPRWTelemetry(set exporter.CreateSettings) prwTelemetry {
+
+	meter := metadata.Meter(set.TelemetrySettings)
+	// TODO: create helper functions similar to the processor helper: BuildCustomMetricName
+	prefix := "exporter/" + metadata.Type + "/"
+
+	failedTranslations, _ := meter.Int64Counter(prefix+"failed_translations",
+		metric.WithDescription("Number of translation operations that failed to translate metrics from OTEL to Prometheus"),
+		metric.WithUnit("1"),
+	)
+
+	translatedTimeSeries, _ := meter.Int64Counter(prefix+"translated_metrics",
+		metric.WithDescription("Number of Prometheus time series that were translated from OTEL metrics"),
+		metric.WithUnit("1"),
+	)
```

Comment:
Should we be adding[ the unit suffixes ](https://prometheus.io/docs/practices/naming/)  to the metric name even though metric normalization may add them at the prometheus exporter?

Reply:
This is an Otel counter, therefore I don't think this requirement should apply.